bugsnag-android-core/src/test/java/com/bugsnag/android/EventRedactionTest.kt

```diff
@@ -13,7 +13,8 @@ internal class EventRedactionTest {
     fun testEventRedaction() {
         val event = Event(
             null,
-            generateImmutableConfig(),
+            generateImmutableConfig()
+                .copy(redactedKeys = setOf("password", "changeme")),
```

Comment:
Is there still a test case in the codebase which asserts that the default value of `redactedKeys` equals "password"? If not, then it'd probably be worth adding one.

Reply:
There are in other tests, but they're testing `JsonStream` and such. I'll add another test to `EventRedactionTest` for the default "password" key.